amads/melody/gradus.py

```diff
@@ -0,0 +1,142 @@
+__author__ = "Yiwen Zhao"
+
+import numpy as np
+from ..core.basics import Score, Note, Part
+
+def calculate_gradus(score: Score) -> float:
```

Comment:
Shorten functions like `calculate_` to `get_` to save typing?

Reply:
+1